File: Numbers/fibonacci_sequence/fibonacci_sequence.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println("Enter the number you want calculated up to...")
  var loc int
  fmt.Scan(&loc)
  sequence := []int{0, 1}
  for i := 1; i < loc + 1; i++ {
    sum := sequence[i] + sequence[i-1]
    sequence = append(sequence, sum)
  }
  fmt.Println(sequence)
}

File: Numbers/fibonacci_sequence/fibonacci_sequence.js
//fibonacci sequence in javascript 
let readlineSync = require("readline-sync");

let loc = parseInt(readlineSync.question("Enter number you want to go up to: "));
let fibonacci = [0, 1];

for (let i = 2; i <= loc; i++) {
    fibonacci[i] = fibonacci[i - 1] + fibonacci[i - 2];
}

console.log(fibonacci);
